### ISSUE TITLE
:item_wrapper_tag and :nested boolean style

### ISSUE THREAD

**Issue by janko** [heartcombo/simple_form#926]
I'm using SimpleForm with Twitter Bootstrap, so I'm using the "nested" boolean form. However, I'm running into issues when I want to set `:item_wrapper_tag` with `f.input :attribute, as: :radio_buttons/check_boxes`. The issue being that I can't set it.

Now, I've read through all the issues about this, and one of the maintainers said that the reason for this is that `<label>` is already acting as the `:item_wrapper_tag`. But I want the following HTML:

``` html
<ul>
  <li>
    <label>
      <input type="radio" name="..." value="...">
      "Some label"
    </label>
  </li>
</ul>
```

In other words, could the `<label>` be a part of the _input_, rather than acting as the `:item_wrapper_tag`, so that we could additionally set the `:item_wrapper_tag` to `:li` or something else?
